src/main/resources/xslt/outputs/odt/models.xsl

```diff
@@ -1,544 +1,641 @@
-				
-				<text:p text:style-name="Instruction"><xsl:value-of select="$instructionLabel"/></text:p>
-				<!--<text:p text:style-name="Instruction"><xsl:value-of select="$instructionFormatMaj"/></text:p>-->
-				
-			</xsl:when>
-		</xsl:choose>
-		
-		<!-- Go to the children -->
-		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
-			<xsl:with-param name="driver" select="." tunnel="yes"/>
-		</xsl:apply-templates>
-		
-	</xsl:template>
-
-	<xsl:template match="CalculatedVariable" mode="model">
-		<xsl:param name="source-context" as="item()" tunnel="yes"/>
-		<xsl:variable name="languages" select="enoodt:get-form-languages($source-context)" as="xs:string +"/>
-		<xsl:variable name="variableCalculation" select="enoodt:get-calculate($source-context)"/>
-		<xsl:variable name="variableCalculationLabel" select="enoodt:get-calculate-text($source-context,$languages[1],'label')"/>
-		<xsl:variable name="variableCalculationAlert" select="enoodt:get-calculate-text($source-context,$languages[1],'alert')"/>
-		
-		<text:p><xsl:value-of select="$variableCalculation"/></text:p>
-		<text:p><xsl:value-of select="$variableCalculationLabel"/></text:p>
-		<text:p><xsl:value-of select="$variableCalculationAlert"/></text:p>
-		<!-- Go to the children -->
-		<xsl:apply-templates select="eno:child-fields($source-context)" mode="source">
-			<xsl:with-param name="driver" select="." tunnel="yes"/>
-		</xsl:apply-templates>		
-	</xsl:template>
-	
-</xsl:stylesheet>
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+	xmlns:xs="http://www.w3.org/2001/XMLSchema" 
+	xmlns:fn="http://www.w3.org/2005/xpath-functions" 
+	xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" 
+	xmlns:eno="http://xml.insee.fr/apps/eno" 
+	xmlns:enoodt="http://xml.insee.fr/apps/eno/out/odt" 
+	xmlns:office="urn:oasis:names:tc:opendocument:xmlns:office:1.0" 
+	xmlns:style="urn:oasis:names:tc:opendocument:xmlns:style:1.0" 
+	xmlns:text="urn:oasis:names:tc:opendocument:xmlns:text:1.0" 
+	xmlns:table="urn:oasis:names:tc:opendocument:xmlns:table:1.0"
+	xmlns:fo="urn:oasis:names:tc:opendocument:xmlns:xsl-fo-compatible:1.0" 
+	xmlns:svg="urn:oasis:names:tc:opendocument:xmlns:svg-compatible:1.0" 
+	exclude-result-prefixes="xs fn xd eno enoodt" version="2.0">
+	
+	<xsl:import href="../../../styles/style.xsl"/>
+	
+	<xsl:param name="properties-file"/>
+	<xsl:param name="parameters-file"/>
+	<xsl:param name="labels-folder"/>
+	
+	<xsl:variable name="properties" select="doc($properties-file)"/>
+	
+	<xsl:variable name="header-content" select="doc($properties//HeaderFile)"/>
+	
+	<xd:doc scope="stylesheet">
+		<xd:desc>
+			<xd:p>An xslt stylesheet who transforms an input into Odt through generic driver templates.</xd:p>
+			<xd:p>The real input is mapped with the drivers.</xd:p>
+		</xd:desc>
+	</xd:doc>
+	
+	<!-- Forces the traversal of the whole driver tree. Must be present once in the transformation. -->

```

Comment:
This comment applied on the template not the variable, isn't it ?

Reply:
yes, of course,I will move this comment